new/collapse-mixlib-install-in-tk.md

```diff
@@ -0,0 +1,171 @@
+
+#### Provisioner setting require_chef_omnibus use cases
+These are the values for require_chef_omnibus and their intent.
+* `true`: If "chef" is installed don't install anything, otherwise install latest
+* `false`: Don't install anything
+* `latest`: Always install latest version
+* `<version>`: If "chef" is installed don't install anything, otherwise install specifed version. Accepts partial versions.
+
+#### Replacement settings for feature parity
+To improve the user experience and align more closley with the new mixlib-install API I am proposing we replace `require_chef_omnibus` with two new settings: `product_version` and `install_strategy`. `product_version` has already been implemented along with `product_name` and `channel`.
+
+##### New Settings
+* `product_version`: Manage `latest` and `<version>` values.  Supports partial versions.
+* `install_strategy`: `once`, `skip`, `always`. Default: `once`
+  * `once`: Behaves similar to true. Install `product_name` `product_version` once. Don't install `product_name` if detected.
+  * `skip`: Behaves similar to false. Do not install `product_name`.
+  * `always`: Behaves similar to latest. Always install specified/latest `product_name` `product_version`.
+
+###### Examples
+```
+require_chef_omnibus: true
+```
+equivalent to:
+```
+product_version: latest
+install_strategy: once
+```
+_Default settings_
+
+----
+
+```
+require_chef_omnibus: false
+```
+equivalent to:
+```
+install_strategy: skip
+```
+
+----
+
+```
+require_chef_omnibus: latest
+```
+equivalent to:
+```
+product_version: latest
+install_strategy: always
+```
+
+----
+
+```
+require_chef_omnibus: 12
+```
+equivalent to:
+```
+product_version: 12
+install_strategy: once
+```
+
+###### Additional Flexibility
+```
+product_version: 12
+install_strategy: always
+```
+
+##### Future
+We could add an upgrade install strategy which only installs the package if a newer package is available.
+
+###### Examples
+```
+product_version: latest
+install_strategy: upgrade
+```
+
+----
+
+```
+product_version: 12.19
+install_strategy: upgrade
+```
+
+#### Opportunity for improvement
+Although adding install_strategy will provide feature parity, we should consider what features we really need in Kitchen moving forward. We may not need this option if we come up with the use cases we feel Kitchen should support moving forward.
+Does Kitchen need to support the same require_chef_omnibus configurabilty with the new API?
+
+By simplifying the use cases, can we automate the install and upgrade paths without introducing a new setting?
+
+#### Mixlib Install changes
+A new script param, `install_strategy`, will be added to `install.sh` and `install.ps1`. The option will determine how the install functions will install the package. The script params will default to always which is the current behavior.
```

Comment:
"default to always which" -> "default to always, which". Consider rewording to something like "The default params will be set to maintain the current behavior of always installing the specified package."